apps/docs/src/app/core/component-docs/toolbar/examples/toolbar-overflow-example.component.html

```diff
@@ -0,0 +1,37 @@
+<fd-toolbar [shouldOverflow]="true">
+    <label fd-toolbar-item></label>
```

Comment:
i think this can be removed

Reply:
fd-toolbar-item is needed because in overflow mode I query for all fd-toolbar-items

Reply:
There are other toolbar items, I'm confused, why do you need an empty one?

Reply:
ah right this one can be removed, I'll do it tomorrow